src/site/content/en/metrics/lcp/index.md

```diff
@@ -4,7 +4,7 @@ title: Largest Contentful Paint (LCP)
 authors:
   - philipwalton
```

Comment:
@philipwalton, should we keep this to one author? Not sure what the threshold is for adding a byline when you add content.